services/operator_info.go

```diff
@@ -0,0 +1,134 @@
+package services
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/Rocket-Rescue-Node/credentials"
+	"github.com/Rocket-Rescue-Node/credentials/pb"
+	"github.com/Rocket-Rescue-Node/rescue-api/models"
+	authz "github.com/Rocket-Rescue-Node/rescue-api/models/authorization"
+	"github.com/Rocket-Rescue-Node/rescue-api/util"
+	"go.uber.org/zap"
+)
+
+const (
+	// The maximum age for a operator info request to be considered valid.
+	operatorInfoRequestMaxAge = time.Duration(15) * time.Minute
```

Comment:
let's just reuse the CredentialRequestMaxAge here. Both are "how long should we consider a signed message from the node as admissible" and I don't see a reason we'd want the granularity to configure them differently for OperatorInfo vs Credential